meetings/2021-03-29.md

```diff
@@ -0,0 +1,82 @@
+# 2021-03-29 Authentication Panel
+
+https://meet.jit.si/solid-authentication
+
+## Agenda
+
+* [meeting notes 2021-03-22 #154](https://github.com/solid/authentication-panel/pull/154) (1m)
+* [Define Solid-OIDC requirements for what client description WebID Document should include #75](https://github.com/solid/authentication-panel/issues/75)
+* [support DID-JWT?](https://github.com/solid/authentication-panel/issues/157)
+* [ontology security](https://github.com/solid/authentication-panel/issues/156)
+
+
+## Present
+
+* Dmitri
+* Aaron Coburn
+* Matthieu
+* Henry Story
+* Elf Pavlik
+* CSarven
+
+## Minutes
+
+### Define Solid OIDC issue
+ [Define Solid-OIDC requirements for what client description WebID Document should include #75](https://github.com/solid/authentication-panel/issues/75)
+
+Elf: by whome is the data used?
+
+Aaron: An Application identifier may support for example not support  authentication identifiers. If the data that is entered in to the authorization endpoint is not include in the client identifier data set then the request fails.
+
+Ultimately the server has allow lists for these values. If the server says it does not support implicit flow, but the flow is not included in the allow list.
```

Comment:
Something seems to be missing from the second sentence below -- i.e., there's no "then" clause following the "but" clause...
```suggestion
Ultimately, the server has "allow" lists for these values. If the server says it does not support implicit flow, but the flow is not included in the allow list, then _____ .
```